addons/mass_mailing/wizard/mailing_mailing_schedule_date.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailingMailingScheduleDate(models.TransientModel):
+    _name = "mailing.mailing.schedule.date"
```

Comment:
As we are introducing this wizard back, could be great to add some tests with it just to cover its code, even if it is simple :) .

Reply:
I have added some tests, you can have a look :)